File: College-Sports-WebApp/ClientApp/src/api/models/geo-broadcast.ts
/* tslint:disable */
/* eslint-disable */
import { Market } from '../models/market';
import { Media } from '../models/media';
import { Type } from '../models/type';
export interface GeoBroadcast {
  id?: number;
  lang?: string | null;
  market?: Market;
  media?: Media;
  region?: string | null;
  type?: Type;
}

File: College-Sports-WebApp/Database/Models/GeoBroadcast.cs
namespace College_Sports_WebApp.Database.Models{ 

    public class GeoBroadcast
    {
        public Type type { get; set; }
        public Market market { get; set; }
        public Media media { get; set; }
        public string lang { get; set; }
        public string region { get; set; }
    }

}